docs/src/blockchain-development/calling_contracts.md

```diff
@@ -0,0 +1,127 @@
+# Calling Contracts
+
+End users and smart contracts typically need to communicate with each other.
+
+The FuelVM achieves end user and inter contract communication primarily with the [`call`](https://github.com/FuelLabs/fuel-specs/blob/master/specs/vm/opcodes.md#call-call-contract) opcode.
+
+Sway provides a nice way manage calls with its `abi` system.
+
+## Script Calls
+
+Here is an example of a *script calling a contract* in Sway:
+
+```rs
+script;
+
+abi MyContract {
+    fn foo(field_1: bool, field_2: u64);
+}
+
+const contract_address = 0x79fa8779bed2f36c3581d01c79df8da45eee09fac1fd76a5a656e16326317ef0;
+
+fn main() {
+    let x = abi(MyContract, contract_address);
+    x.foo(true, 3);
+}
+```
+
+## Inter Contract Calls
```

Comment:
This section is basically a duplicate of the previous one. There's no real difference between calling _from_ a script or a contract, so this should simply be mentioned in a single sentence instead of duplicating an entire section.

Reply:
Agreed, we can merge, I just wanted to showcase that it works in both. I'll make a note.

Reply:
Removed.